security/v1/types.go

```diff
@@ -96,6 +96,26 @@ type SecurityContextConstraints struct {
 	// used to generate a value for a pod the first non-wildcard profile will be used as
 	// the default.
 	SeccompProfiles []string `json:"seccompProfiles,omitempty" protobuf:"bytes,20,opt,name=seccompProfiles"`
+
+	// AllowedUnsafeSysctls is a list of explicitly allowed unsafe sysctls, defaults to none.
+	// Each entry is either a plain sysctl name or ends in "*" in which case it is considered
+	// as a prefix of allowed sysctls. Single * means all unsafe sysctls are allowed.
+	// Kubelet has to whitelist all allowed unsafe sysctls explicitly to avoid rejection.
```

Comment:
@openshift/api-review Should we modify the `privileged` SCC to allow all unsafe sysctls by default (like we did for allowedCapabilities: https://github.com/openshift/origin/issues/12741)? In this case, we won't have a confusion when a cluster admin didn't have permissions for using usafe sysctls.

Reply:
that's what I would expect